spec/factories/matches.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :match do
+    sequence(:game_name) { |n| "football-#{n}" }
```

Comment:
use faker gem avoid using this #n namer (this applies to al all factories files)